LayoutTests/fast/text/accessibility-bold-system-font-3-expected-mismatch.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html><!-- webkit-test-runner [ runSingly=true ] -->
+<html>
+<head>
+<script>
+if (window.internals) {
+    window.internals.invalidateFontCache();
```

Comment:
Feels odd for a test to have to worry about caches. Why doesn't `window.internals.settings.setShouldMockBoldSystemFontForAccessibility` invalidate the cache?

Reply:
Good idea!